Source/settings/items.ts

```diff
@@ -10,6 +10,9 @@ module FullScreenPokemon {
             "gameStarted": {
                 "valueDefault": false
             },
+            "collectionKeys": {
+                "valueDefault": []
+            },
```

Comment:
I'm moving the logic of `collecitonKeys` to StateHolder but I need a way to reset `collectionsKeys` when ItemsHolder is cleared. I haven't thought of a better way than what I had before (`StateHolder.clearCollectionKeys`)